tests/e2e/runner/.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
Are we ever running these tests with non-local env vars? If not maybe we want a committed test env file?

Reply:
(see resolved comment in https://github.com/ArtBlocks/artblocks-subgraph/pull/223#discussion_r1251182993)